resources/user.py

```diff
@@ -48,9 +48,10 @@ def get(self, user_id):
         user_info = user.json()
 
         if user.role == 'property-manager':
-            user_info['properties'], tenants_ids = zip(*((p.json(), p.tenants) for p in PropertyModel.find_by_manager(user_id) if p))
-            tenants_list = [TenantModel.find_by_id(t) for t in set(tenants_ids)]
-            user_info['tenants'] = [t.json() for t in tenants_list if t]
+            user_info['properties'], tenants_ids = zip(*((p.json(), p.tenantIDs) for p in PropertyModel.find_by_manager(user_id) if p))
```

Comment:
I'm getting the error: `AttributeError: 'PropertyModel' object has no attribute 'tenantIDs'` when trying to do this. Which is confusing, because `tenantIDs` definitely exists on the PropertyModel. @htharker42 @Chris-Boe do you have any suggestions or thoughts on what I might be doing wrong?

Reply:
Figured this out, please disregard. Sorry for the noise!